js/src/components/paid-ads/constants.js

```diff
@@ -0,0 +1,282 @@
+		),
+		lowercaseSingularName: _x(
+			'headline',
+			'Singular and lowercase asset field name',
+			'google-listings-and-ads'
+		),
+		lowercasePluralName: _x(
+			'headlines',
+			'Plural and lowercase asset field name',
+			'google-listings-and-ads'
+		),
+		help: __(
+			'The headline is the first line of your ad and is most likely the first thing people notice, so consider including words that people may have entered in their Google search.',
+			'google-listings-and-ads'
+		),
+	},
+	{
+		key: 'long_headline',
+		min: 1,
+		max: 5,
+		maxCharacterCounts: 90,
+		heading: _x(
+			'Long headlines',
+			'Plural asset field name as the heading',
+			'google-listings-and-ads'
+		),
+		addButtonText: __( 'Add long headline', 'google-listings-and-ads' ),
+		capitalizedName: _x(
+			'Long headline',
+			'Capitalized asset field name as the placeholder or the start of an error message',
+			'google-listings-and-ads'
+		),
+		lowercaseSingularName: _x(
+			'long headline',
+			'Singular and lowercase asset field name',
+			'google-listings-and-ads'
+		),
+		lowercasePluralName: _x(
+			'long headlines',
+			'Plural and lowercase asset field name',
+			'google-listings-and-ads'
+		),
+		help: (
+			<>
+				<p>
+					{ __(
+						'The long headline is the first line of your ad, and appears instead of your short headline in larger ads. Long headlines can be up to 90 characters, and may appear with or without your description.',
+						'google-listings-and-ads'
+					) }
+				</p>
+				<p>
+					{ __(
+						'The length of the rendered headline will depend on the site it appears on. If shortened, it will end with an ellipsis(…).',
+						'google-listings-and-ads'
+					) }
+				</p>
+			</>
+		),
+	},
+
+	{
+		key: 'description',
+		min: 2,
+		max: 5,
+		maxCharacterCounts: [ 60, 90, 90, 90, 90 ],
+		heading: _x(
+			'Descriptions',
+			'Plural asset field name as the heading',
+			'google-listings-and-ads'
+		),
+		addButtonText: __( 'Add description', 'google-listings-and-ads' ),
+		capitalizedName: _x(
+			'Description',
+			'Capitalized asset field name as the placeholder or the start of an error message',
+			'google-listings-and-ads'
+		),
+		lowercaseSingularName: _x(
+			'description',
+			'Singular and lowercase asset field name',
+			'google-listings-and-ads'
+		),
+		lowercasePluralName: _x(
+			'descriptions',
+			'Plural and lowercase asset field name',
+			'google-listings-and-ads'
+		),
+		help: (
+			<>
+				<p>
+					{ __(
+						'The description adds to the headline and provides additional context or details. It can be up to 90 characters, and may appear after the headline.',
+						'google-listings-and-ads'
+					) }
+				</p>
+				<p>
+					{ __(
+						`The length of the rendered description will depend on the site it appears on. If it's shortened, it will end with an ellipsis(…). The description doesn't show in all sizes and formats.`,
+						'google-listings-and-ads'
+					) }
+				</p>
+			</>
+		),
+	},
+];
+
+{
```

Comment:
❓  Why we need to wrap this with `{}`

Reply:
This scoped block is to signal that dynamic initialization is handled here, and that the functions inside are for initialization purposes only and are not expected to be exposed outside this module. I added this as the code comment in https://github.com/woocommerce/google-listings-and-ads/pull/1863/commits/de0abb4063db89f7db3caa396cdee86cb057c711.